lib/pure/sugar.nim

```diff
@@ -238,3 +238,17 @@ macro distinctBase*(T: typedesc): untyped =
   while typeSym.typeKind == ntyDistinct:
     typeSym = getTypeImpl(typeSym)[0]
   typeSym.freshIdentNodes
+
+when defined(nimHasAlias):
+  proc aliasImpl[T1, T2](name: T1, expr: T2) {.magic: "Alias".}
+
+  template `:=`*(name, expr) =
```

Comment:
Unrestricted templates with such generic names like this one did cause a lot of trouble in the past. Most prominent example was the ``[]`` operator for list comprehension. I am sorry, but you have to put it either in it's own module so that you won't accidentally import it, or you have to rename the template to something that is much less likely to clash.

Reply:
done. I've changed this to: `alias: foo = bar`, as was also suggested in other comments